lib/redux/actions/sync.js

```diff
@@ -492,6 +507,33 @@ export function updateProfileFailure(err) {
 }
 
 /*
+ * relating to createCustodialAccount
+ */
+
+export function createCustodialAccountRequest() {
+  return {
+    type: actionTypes.CREATE_CUSTODIAL_ACCOUNT_REQUEST,
+    meta: {source: actionSources[actionTypes.CREATE_CUSTODIAL_ACCOUNT_REQUEST]}
+  };
+}
+
+export function createCustodialAccountSuccess() {
```

Comment:
Commented above on the lack of a payload here...I find that pretty strange in a success action. Would like to understand a bit more why the refetching is necessary instead of just updating the state using the response, which could be put into a `payload`?